pkg/chart/chart.go

```diff
@@ -705,7 +705,7 @@ func (t *Testing) ComputeChangedChartDirectories() ([]string, error) {
 				changedChartDirs = append(changedChartDirs, chartDir)
 			}
 		} else {
-			fmt.Printf("Directory '%s' is no chart directory. Skipping...", chartDir)
+			fmt.Printf("Directory '%s' is not a valid chart directory. Skipping...\n", dir)
```

Comment:
When the tool found non-chart files changed, it output as:
```
Directory '' is no chart directory. Skipping...Directory '' is no chart directory. Skipping...Directory '' is no chart directory. Skipping...Directory '' is no chart directory. Skipping...
```
This was extremely confusing.